src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -270,12 +310,43 @@ def get(self, request: Request, project: Project) -> Response:
                     )
                     and project.organization.flags.codecov_access
                 )
+
+                # Get commit sha from Git blame
+                commit_sha = (
+                    ctx["commit_id"]
+                    if ctx["commit_id"]
+                    else current_config["config"]["defaultBranch"]
+                )
+                try:
+                    should_get_commit_sha = should_get_codecov_data and (
```

Comment:
If we're going to put it behind `should_get_codecov_data` we should move it under `if should_get_codecov_data:`